nodes/geometry/bmesh_tesselate.py

```diff
@@ -0,0 +1,43 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types.node import AnimationNode
+
+class BMeshTessellateNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_BMeshTessellateNode"
+    bl_label = "Triangulate BMesh"
+    bl_width_default = 160
+    searchLabels = ["Tesselate BMesh"]
+
+    quad = IntProperty(name = "Quad Method", update = propertyChanged, max = 3, min = 0)
```

Comment:
they kinda switch the tris, but it's not clear to me how
I was wondering if I should keep them for advanced, I mean how often are they gonna be used?